microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Understand the heap snapshot file format
+description: How the heap snapshot files are structured, when exported from the Memory tool
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 9/16/2022
+---
+# Understand the heap snapshot file format
```

Comment:
```suggestion
# The .heapsnapshot file format
```
If change title, change all spots:
the title: yaml field above
h1 here
toc.yml
link text of links from other pages to here